Wireframe/index.html

```diff
@@ -3,11 +3,47 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <title>WireFrame</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+
   </head>
   <body>
+    <div class="content">
+
+      <header>
+        <h1>Learn Git</h1>
+        <p>Master Git to manage your code, collaborate with teams, and streamline your development process.</p>
+      </header>
+
+      <main class="container">
+
+        <section>
+          <h2>what is git?</h2>
+          <p>Git is a version control system that helps developers track changes in code, collaborate easily, and manage project versions. It allows branching, committing changes, and reverting to earlier versions. Popular platforms like GitHub use Git for smooth teamwork on coding projects.</p>
+          <a href="#" class="read-btn">Read More</a>
+        </section>
+
+        <section class="column">
+          <img src="image/img2.png" alt="git-developer">
```

Comment:
Suggestion: when adding pictures, putting a good name on it, makes the code cleaner . Another suggestion is to use "alt" to describe the picture, like this:` <img src="image/img2.png" alt="Git Developer">`